### DISCUSSION TITLE
need help! what's version choosed for angularJS and nodejs?

### DISCUSSION THREAD

**Discussion by qiangkeming** [dcm4che/dcm4chee-arc-light#3523] [Q&A]
hi, teacher:
       i download dcm4chee-arc-light.git version(5.25.1) source to complie in my computer,   i user interlliJ idea 2910 , the java project can execute in idea , but the web page use angularJs can not to running, i want to kown what version used for angularJs and nodejs?
       tks a lot !

                     from: china kevin
                      2022.1.20

**Comment by shral** [comment]
Hi, 
if you don't need to change anything in the UI, than you can build the whole project by navigating in `dcm4chee-arc-light` folder and calling `mvn install -D secure=all` in the terminal, that will generate an ear file in `dcm4chee-arc-light/dcm4chee-arc-ear/target/` that you should deploy in the WildFly (for example by copping the ear in `/var/local/dcm4chee-arc/wildfly/deployments/`.

If you navigate to the folder `dcm4chee-arc-light/dcm4chee-arc-ui2/package.json` you can see that the UI is currently using `angular:10.1.6`.
Other support tools: 
node is verison: v12.13.0
npm version: 7.6.3
yarn: 1.21.1
